### ISSUE TITLE
how to feed extra data to specific bijector of TransformedDistribution's bijectors?

### ISSUE THREAD

**Issue by gitlabspy** [tensorflow/probability#998]
`x2` is some label informations for `x1`, but how can I pass `x2` (as a constant) into `bi` in `tfb.Chain([bi1, bi2, bi3])`?
```
x1 = tf.random.normal((batch, 32, 32, 3))
x2 = tf.random.normal((batch, 32,32, 3))
trans_dit = tfd.TransformedDistribution(bijectors=tfb.Chain([bi1, bi2, bi3]),...)
loss = trans_dit.log_prob(x1)
with tf.gradientTape() as tape:
     # update...
```
